src/pages/dataTableUtilities/columns.js

```diff
@@ -181,6 +190,32 @@ const COLUMNS = () => ({
     title: tableStrings.selected,
     alignText: 'center',
   },
+  difference: {
+    key: 'difference',
+    title: tableStrings.difference,
+    sortable: true,
+    alignText: 'right',
+  },
+  countedTotalQuantity: {
+    type: 'editable',
+    key: 'countedTotalQuantity',
+    title: tableStrings.actual_quantity,
+    sortable: true,
+    alignText: 'right',
+  },
+  snapshotTotalQuantity: {
+    key: 'snapshotTotalQuantity',
+    title: tableStrings.snapshot_quantity,
+    sortable: true,
+    alignText: 'right',
+  },
+  modalControl: {
```

Comment:
should just be `batchControl` or something similar, since it uses the `BATCHES` title